ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.13.yaml

```diff
@@ -250,13 +250,13 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-vsphere-ipi-proxy-workers-rhel8
-- as: vsphere-upi-f28
+- as: vsphere-upi-encrypt-f28
```

Comment:
In order not make my pr conflict, please don't change this job, I will update it in my pr: https://github.com/openshift/release/pull/54732/files

Reply:
Hi @JianLi-RH It's very common to address conflict in CI. I personally don't think it's a good reason to ask author to rework their PR due to upcoming conflict. It's not that difficult to tackle conflict. If you have any difficulty with it, please let me know. we can work it out together.

Reply:
@JianLi-RH I fully understand your concern. 
During reviewing your PR, found several duplicated copies, so I prepare this PR to sync the ci profiles. 
And I saw you opened several PRs to add profiles in all kinds of upgrade path, that might bring some more mismatch or duplicated ones.
I also agree with yangyang, it is very common when team is working on a central repo to update some shared files, we always need to resolve conflicts. 
If I only need to wait the only PR - https://github.com/openshift/release/pull/54732 get merged, I can wait for it, and rebase mine after yours get merged.